frontend/models/product-list/index.ts

```diff
@@ -66,7 +66,7 @@ export async function findProductList(
          : null);
    const title =
       productList?.title ??
-      (deviceWiki?.title ? deviceWiki?.title + ' Parts' : '');
```

Comment:
Man, maybe I messed up with our API but I meant to keep the field names the same. Maybe I just mis-read the code when I tried to extract which fields where used.

Let's validate that this still makes sense:
[`deviceTitle` is set to `$blurb->wiki_title`](https://github.com/iFixit/ifixit/blob/master/iFixit/Cart/Controllers/PartCollectionsDeviceApiController.php#L73)
[`$blurb->wiki_title` is set from the `wikis.title` table](https://github.com/iFixit/ifixit/blob/3618dbdea19912b0a7c57ae4ca278c2523ac4fd0/Guide/Objects/Blurb.php#L646)

Previously, this was set to ``deviceWiki.title` which [comes from: `$wiki->title`](https://github.com/iFixit/ifixit/blob/3618dbdea19912b0a7c57ae4ca278c2523ac4fd0/Guide/Libs/GuideAPILib_2_0.php#L1515) which is the title from the `title` DB column.

So we're good and I think I just accidentally change the name of this field (I think I confused this line for line 54 above).

Reply:
I think this is fine